.github/workflows/deployment.yml

```diff
@@ -17,18 +17,19 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
+      - uses: actions/checkout@v3
```

Comment:
Fun fact! This is why it wasn't working...I didn't actually check out the repo in the workflow.